source/stylesheets/refills/_vertical-tabs.scss

```diff
@@ -14,7 +14,7 @@
   $tab-content-background: lighten($tab-border-color, 10%);
   $tab-active-color: $tab-content-background;
   $tab-inactive-color: $base-background-color;
-  $vertical-tabs-height: em(300);
+  $vertical-tabs-height: 18.75em;
```

Comment:
Totally agree.

For now, I didn’t want to introduce any visual changes so I did a direct conversion. Eventually, I believe Refills should strive to not declare fixed sizes and leverage a more flexible system by using flexbox and other means.